plugins/media.js

```diff
@@ -73,3 +73,7 @@ export function isRichMedia(media) {
     isIIIFMedia(media) ||
     isPlayableMedia(media);
 }
+
+export function requiresDashJS(media) {
```

Comment:
Could you add a unit test?
Should be straight forward, there are already similar ones in `tests/unit/plugins/media.spec.js` for the rest of the methods in here.

Reply:
sure